requirements.in

```diff
@@ -65,7 +65,3 @@ locust   # Used in tools/load_test_locustfile.py
 
 # Documentation
 sphinx
-
-# Automated spreadsheet report generation (e.g., for consent validation metrics reporting)
-pandas
-gspread
```

Comment:
It seems like 'requirements.in' is unmaintained now and should be removed from the project.

Reply:
I added comments to requirements.in to clarify that it is not currently being used;  in case there is the potential to go back to using pip-tools / pip-sync, I thought I would keep the file around for future reference.